trollmoves/dispatcher.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2019
+#
+# Author(s):
+#
+#   Martin Raspaud <martin.raspaud@smhi.se>
+#   Panu Lahtinen <panu.lahtinen@fmi.fi>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Dispatcher dispatches(!) files to some destination.
+
+It listens to posttroll messages to find out what to dispatch.
+
+
+Format of the configuration file
+================================
+
+.. highlight:: yaml
+
+Example config::
+
+    client1:
+      host: ftp://ftp.client1.com
+      connection_parameters:
+        connection_uptime: 60
+      filepattern: '{platform_name}_{start_time}.{format}'
+      directory: /input_data/{sensor}
+      aliases:
+        product:
+          natural_color: dnc
+          overview: ovw
+      dispatch:
+        - topics:
```

Comment:
Is there a need to match other things, like originating host or `service`? Looking at the very generic example topics, I could see a need to pre-filter the messages with something more, e.g. when there are development streams being tested.

Reply:
It's really up to the user to define what topics they would like to use. Do you want me to give a more complex example ?
Regarding service or host, for me that would be matched with the metadata somehow, but topics should work too.

Reply:
I'll be having a very detailed topics, so I'm fine with this. It was just a thought that someone might want to have only data coming from local reception to be forwarded even if there are other sources for the same products with the same topics.